### PR TITLE
Update cryptography reference - Closes #934

### LINKED ISSUE #934: Update lisk-cryptography reference
Add missing command references

See also: https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-cryptography/src/keys.ts#L158-L179

Parent issue: #845

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #934

### CODE REVIEW

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 1]
```diff
@@ -1004,9 +1227,54 @@ signDataWithPrivateKey(data,privatekey)
...
+[source,js]
+----
+verifyData(data,signature,publicKey)
+----
+
+==== Parameters
+
+. `data`(Buffer): The data to be verified, e.g. a transaction object.
+. `signature`(Buffer): description
+. `publicKey`(Buffer): description
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]: I think description is missing here

**Review by ishantiw — COMMENTED** [review]
LGTM, a comment regarding missing description of parameters but there are other places where the desc of some return values and parameters is missing

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 2]
```diff
@@ -781,6 +666,24 @@ cryptography.getPrivateAndPublicKeyFromPassphrase(
...
+==== Syntax
+
+[source,js]
+----
+verifyChecksum(integerSequence)
+----
+
+==== Parameters
+
+`integerSequence`(number[]): description
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]: `Input for verifyChecksum must be a sequence of integers between 0 and 31.`

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 3]
```diff
@@ -781,6 +666,24 @@ cryptography.getPrivateAndPublicKeyFromPassphrase(
 } */
 ----
 
+=== verifyChecksum
+description
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]: `To verify the checksum of Base32 encoded addresses.`

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 4]
```diff
@@ -1004,9 +999,54 @@ signDataWithPrivateKey(data,privatekey)
...
 
 `Buffer`: Returns the signed data.
+
+=== signData
+
+Alias for <<signDataWithPassphrase>>.
+
+=== verifyData
+
+description
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]: `Verifies that a signature for a given data matches the provided public key.`

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 5]
```diff
@@ -1004,9 +999,54 @@ signDataWithPrivateKey(data,privatekey)
...
+
+[source,js]
+----
+verifyData(data,signature,publicKey)
+----
+
+==== Parameters
+
+. `data`(Buffer): The data to be verified, e.g. a transaction object.
+. `signature`(Buffer): description
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]: `Signature against which data is verified`

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 6]
```diff
@@ -1004,9 +999,54 @@ signDataWithPrivateKey(data,privatekey)
...
+. `publicKey`(Buffer): description
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]: `public key used to sign the data`

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 7]
```diff
@@ -781,6 +666,24 @@ cryptography.getPrivateAndPublicKeyFromPassphrase(
...
+verifyChecksum(integerSequence)
+----
+
+==== Parameters
+
+`integerSequence`(number[]): description
+
+==== Return value
+
+`number[]`: description
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]: I think return value is a boolean whether address was valid or not

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 8]
```diff
@@ -1004,9 +999,54 @@ signDataWithPrivateKey(data,privatekey)
...
+
+==== Parameters
+
+. `data`(Buffer): The data to be verified, e.g. a transaction object.
+. `signature`(Buffer): description
+. `publicKey`(Buffer): description
+
+==== Return value
+
+`Boolean`: Returns the signed data.
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]: Returns true if signed data is valid or else returns false

**Comment by Tschakki** [comment]
Hey @ishantiw  I applied your requested changes :) Please be aware that you can post these comments also as suggestions, then it is more convenient to apply the suggestions.

**Review by ishantiw — APPROVED** [review]
LGTM

**Pull request merged**